### PR TITLE
Fixing pouta ansible demo

### PR DESCRIPTION
**Pull request by jemaltahir**
Updated and tested in cPouta.

### CODE REVIEW

File: demo.yml [hunk 1]
```diff
@@ -10,9 +10,12 @@
 
   vars:
     # These will need changing based on your project
-    demo_key: jenkins
-    demo_sg: demo-sg-jenkins
+    demo_key: ansible-vm-key
+    demo_sg: demo-sg-ansible
     demo_security_groups: default,{{ demo_sg }} # don't add spaces here!
+    net_id: c55bc796-841f-4704-a1a2-8f29bb9a699a
```
Comment (lvarin) [inline demo.yml]: Maybe remove this?

File: demo.yml [hunk 2]
```diff
@@ -10,9 +10,12 @@
...
+    net_name: project_2001316
+    path_to_key: "/home/jtahir/.ssh/id_rsa.pub" # path to local pc public key
```
Comment (lvarin) [inline demo.yml]: Perhaps, leave this empty so it fails if the user does not fill it up. It is a bit more user friendly to use:

 ansible-playbook demo.yml -e path_to_key=.....pub

File: demo.yml [hunk 3]
```diff
@@ -10,9 +10,12 @@
 
   vars:
     # These will need changing based on your project
-    demo_key: jenkins
-    demo_sg: demo-sg-jenkins
+    demo_key: ansible-vm-key
+    demo_sg: demo-sg-ansible
     demo_security_groups: default,{{ demo_sg }} # don't add spaces here!
+    net_id: c55bc796-841f-4704-a1a2-8f29bb9a699a
+    net_name: project_2001316
```
Comment (lvarin) [inline demo.yml]: Perhaps, leave this empty so it fails if the user does not fill it up. It is a bit more user friendly to use:

ansible-playbook demo.yml -e net_name='project_2001316'

File: demo.yml [hunk 4]
```diff
@@ -33,30 +36,46 @@
...
+      os_keypair:
+          state: present
+          name: "{{ demo_key }}"
+          public_key_file: "{{ path_to_key }}"
 
     - name: Create a virtual machine
       register: result
       os_server:
+        state: present
         name: pouta-demo
```
Comment (lvarin) [inline demo.yml]: Use a variable instead, again I find it more user friendly for the non ansible people.

**Review by lvarin — CHANGES_REQUESTED** [review]
Other than that LGTM

File: demo.yml (hunk 1 above)
Comment (jemaltahir) [reply demo.yml]: Thanks for pointing out that i have to describe explicitly.

File: demo.yml (hunk 2 above)
Comment (jemaltahir) [reply demo.yml]: Yes, it is convenient to pass as an environment variables.

**Review by lvarin — APPROVED** [review]
LGTM, thanks for the changes

**Pull request merged**